src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/AssessmentResourceIntTest.kt

```diff
@@ -52,6 +62,14 @@ class AssessmentResourceIntTest : SqsIntegrationTestBase() {
   @Autowired
   private lateinit var offenderRepository: OffenderRepository
 
+  @PersistenceContext
+  lateinit var entityManager: EntityManager
+
+  @AfterEach
+  fun tearDownWithinTransaction() {
+    entityManager.clear()
```

Comment:
why is this required?  presumably this is related to the issues with running the integration tests

Reply:
TO be faire I don't think it is ,... I was ment to remove it,.. will check first!